workflow/JSON-visualizer/index.js

```diff
@@ -0,0 +1,93 @@
+const jsonWrapper = document.querySelector('.json-wrapper');
+const buildTreeButton = document.querySelector('.build-tree-btn');
+
+function isJson(str) {
+  try {
+    JSON.parse(str);
+  } catch (e) {
+    return false;
+  }
+  return true;
+}
+
+function createAndAppendParagraph(parent, innerHtml) {
+  const newParagraph = document.createElement('p');
+  newParagraph.innerHTML = innerHtml;
+  parent.append(newParagraph);
+}
+
+function createAndAppendDivWithClass(parent, className) {
+  const div = document.createElement('div');
+  div.className = className;
+  parent.append(div);
+  return div;
+}
+
+function highlightDifferentDataTypes(data) {
+  switch (typeof data) {
+    case 'number':
+      return 'red';
+
+    case 'string':
+      return 'purple';
+
+    case 'bigint':
+      return 'yellow';
+
+    case 'boolean':
+      return 'green';
+
+    default:
+      return 'blue';
+  }
```

Comment:
`switch-case` is great (remove empty lines after `return` line).

FYI: other useful way to handle things like this - using object:
```
const variants = {
number: "red",
string: "purple",
bigint: "yellow",
boolean: "green",
default: "blue"
}
const type = typeof data;
return variants[type] || variants.default;
```